### PR TITLE
City forecast layout

### PR DESCRIPTION
**Pull request by DmitriyKrivovyaz**

### CODE REVIEW

File: client/src/layouts/cityForecast/CurrentCityForecast/CurrentCityForecast.module.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  align-items: center;
+}
+
+.weatherItem {
+  margin-bottom: 10px;
+}
+
+.city {
+  font-weight: 600;
+  font-size: 64px;
```
Comment (glebdronchenko) [inline client/src/layouts/cityForecast/CurrentCityForecast/CurrentCityForecast.module.css]: Hardcoded value.

File: client/src/layouts/cityForecast/CurrentCityForecast/CurrentCityForecast.module.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+.temperature {
+  font-weight: 500;
+  font-size: 68px;
```
Comment (glebdronchenko) [inline client/src/layouts/cityForecast/CurrentCityForecast/CurrentCityForecast.module.css:29]: Hardcoded value.

File: client/src/layouts/cityForecast/CurrentCityForecast/CurrentCityForecast.module.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  line-height: 83px;
+}
+
+.weatherConditions {
+  font-weight: 300;
+  font-size: 32px;
```
Comment (glebdronchenko) [inline client/src/layouts/cityForecast/CurrentCityForecast/CurrentCityForecast.module.css]: Hardcoded value. Please, check all hardcoded values.

File: client/src/mock/mock.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  temperature: 7,
+  conditions: 'clear',
+  time: '20:00',
+  weekDay: 'Monday',
+  day: 29,
+  month: 'September',
+  year: 2021
+};
+
+export { currentCityData };
```
Comment (glebdronchenko) [inline client/src/mock/mock.js:16]: Did you take this structure from API?

File: client/src/layouts/cityForecast/ExtraCityForecast/ExtraCityForecast.module.css [hunk 5]
```diff
@@ -0,0 +1,13 @@
+.container {
+  width: 302px;
```
Comment (ximet) [inline client/src/layouts/cityForecast/ExtraCityForecast/ExtraCityForecast.module.css]: why 302? why not 302,00000002? Strange 2px for me

File: client/src/App.jsx [hunk 6]
```diff
@@ -27,7 +28,9 @@ function App() {
   return (
     //Use BrowserRouter to use Link from react-router-dom
     <BrowserRouter>
+      <BackgroundImage bgImage={bgImage} />
```
Comment (anton-vlasik) [inline client/src/App.jsx:31]: Why didn't you name your component BgImage? :) 
I mean why not using the whole name for you property?

**Pull request merged**